docs/src/description/appendix.md

```diff
@@ -0,0 +1,7 @@
+# Appendix
+
+Contains additional information about some concepts that are not covered in the main documentation.
```

Comment:
nit: I would make this a full sentence, i.e. "This appendix contains information about ..."